### PR TITLE
[ja] Clarify API version stability and guidelines

### PR DESCRIPTION
**Pull request by ystkfujii**
- I have addressed the fix for https://github.com/kubernetes/website/commit/d308cbb35a335a5eb34ef4a7fb8a20ea1b98d100.
  - However, I gave up on deprecation-polisy.md because it did not exist in the Japanese content.
  - Instead, I created https://github.com/kubernetes/website/issues/39861

### CODE REVIEW

**Comment by ystkfujii** [comment]
Deploy preview:

- https://deploy-preview-39862--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/overview/kubernetes-api/
- https://deploy-preview-39862--kubernetes-io-main-staging.netlify.app/ja/docs/reference/using-api/

File: content/ja/docs/reference/using-api/_index.md [hunk 1]
```diff
@@ -38,30 +38,31 @@ APIのバージョンが異なると、安定性やサポートのレベルも
...
-    スキーマの変更に伴い、APIオブジェクトの削除、編集、再作成が必要になる場合があります。
-    編集作業は単純ではないかもしれません。
+    後続のbetaまたはstable APIバージョンに適応することはAPIオブジェクトの編集や再作成が必要になる場合があり、単純ではないかもしれません。
     移行に伴い、その機能に依存しているアプリケーションのダウンタイムが必要になる場合があります。
 
   - 本番環境での使用は推奨しません。
-    後続のリリース は、互換性のない変更を導入する可能性があります。
-    独立してアップグレード可能な複数のクラスターがある場合、この制限を緩和できる可能性があります。
+    後続のリリースは、互換性のない変更を導入する可能性があります。
+    beta APIが非推奨となり提供されなくなった際には、後続のbetaまたはstable APIバージョンに移行するためにbeta APIバージョンを使用する必要があります。
```
Comment (t-inu) [inline content/ja/docs/reference/using-api/_index.md:65]:
```suggestion
    beta APIを使用する場合、そのbeta APIが非推奨となり提供されなくなった際には、後続のbetaまたはstable APIバージョンに移行する必要があります。
```
「ベータ版を使うのなら、それが非推奨となり提供されなくなった時点で、後続のベータ版か安定版を使いなさい」という意味だと思いました。

File: content/ja/docs/reference/using-api/_index.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ APIのバージョンが異なると、安定性やサポートのレベルも
 
 - Stable:
   - バージョン名は `vX` であり、`X` は整数である。
-  - 安定版の機能は、リリースされたソフトウェアの中で、その後の多くのバージョンに登場します。
+  - stable APIバージョンはKubernetesのメジャーバージョン内の全てのリリースで利用可能であり、stable APIを削除したKubernetesのメジャーバージョンリビジョンの計画は現在ありません。
```
Comment (t-inu) [inline content/ja/docs/reference/using-api/_index.md:74]:
```suggestion
  - stable APIバージョンはKubernetesのメジャーバージョン内の全てのリリースで利用可能であり、stable APIを削除するようなKubernetesのメジャーバージョンの修正は、現在計画されていません。
```
I tried to translate more understandable.

**Review by t-inu — COMMENTED** [review]
I left comments, please check.

File: content/ja/docs/concepts/overview/kubernetes-api.md [hunk 3]
```diff
@@ -156,10 +158,18 @@ Kubernetesプロジェクトは、既存のクライアントとの互換性を
...
+
+beta APIを採用した場合、APIが卒業(Graduate)したら、後続のbetaまたはstable APIに移行する必要があります。
+これを行うのに最適な時期は、オブジェクトが両方のAPIバージョンから同時にアクセスできるbeta APIが非推奨期間です。
+beta APIが非推奨期間を終えて提供されなくなったら、代替APIバージョンを使用する必要があります。
 
 {{< note >}}
-Kubernetesは、 _alpha_ APIバージョンについても互換性の維持に注力しますが、いくつかの事情により不可である場合もあります。アルファAPIバージョンを使っている場合、クラスタのアップグレードやAPIが変更された場合に備えて、Kubernetesのリリースノートを確認してください。
+Kubernetesは、 _alpha_ APIバージョンについても互換性の維持に注力しますが、いくつかの事情により不可である場合もあります。
+alpha APIバージョンを使っている場合、クラスターをアップグレードする時にKubernetesのリリースノートを確認してください。
+アップグレードのために既存のalphaオブジェクトをすべて削除する必要がある互換性の無い方法でAPIが変更される場合があります。
```
[the remark this answers is not in the document]
Comment (ystkfujii) [reply content/ja/docs/concepts/overview/kubernetes-api.md]: that is better !

**Review by ystkfujii — COMMENTED** [review]
@t-inu 

Thank you for your comment.
I applied suggestions !

**Review by t-inu — COMMENTED** [review]
Thank you for adopting my suggestions.
/lgtm

**Comment by atoato88** [comment]
LGTM :smile:

**Comment by ystkfujii** [comment]
/assign @nasa9084
Could you review this?

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**